index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Embedding images and videos</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<nav id="navbar">
+	<ul>
+		<li><a href="#"><img  src="menu.PNG"></a></li>
```

Comment:
An `img` element must have an `alt` attribute that is to be rendered when the element to which it is applied cannot be rendered.